libraries/AP_Motors/examples/AP_Motors_test/AP_Motors_test.cpp

```diff
@@ -280,74 +286,24 @@ void loop()
     }
 }
 
+bool first_layout = true;
+
 // print motor layout for all frame types in json format
-void print_all_motor_matrix()
+void print_all_motors()
 {
     hal.console->printf("{\n");
     hal.console->printf("\t\"Version\": \"%s\",\n", VERSION);
     hal.console->printf("\t\"layouts\": [\n");
 
-    bool first_layout = true;
-    char frame_and_type_string[30];
+    first_layout = true;
 
     for (uint8_t frame_class=0; frame_class <= AP_Motors::MOTOR_FRAME_DECA; frame_class++) {
         for (uint8_t frame_type=0; frame_type < AP_Motors::MOTOR_FRAME_TYPE_Y4; frame_type++) {
-            if (frame_type == AP_Motors::MOTOR_FRAME_TYPE_VTAIL ||
```

Comment:
I was skipping these because its a bit un-clear how they should be drawn. The original idea was to generate a diagram. https://discuss.ardupilot.org/t/copter-wiki-motor-diagram-overhaul/101046/24